src/defaults.js

```diff
@@ -117,6 +117,26 @@ var bookingInstant = {
 
 };
 
+// Preset: bookingGraph = 'instant_payment'
+var bookingInstantPayment = {
+
+  timekitCreateBooking: {
+    graph: 'instant_payment',
+    action: 'tentative',
+    event: {
+      invite: true,
+      my_rsvp: 'accepted',
+      sync_provider: true
```

Comment:
Waiting for the sync allows library users to get the event ID in the response. I won't change this since it's been as a default for ages and could break existing integrations.